### PR TITLE
Show SRPM link only if present

### PR DESCRIPTION
**Pull request by mfocko**
Signed-off-by: Matej Focko <mfocko@redhat.com>

### CODE REVIEW

File: frontend/components/results_srpm.js [hunk 1]
```diff
@@ -60,6 +60,14 @@ const ResultsPageSRPM = (props) => {
         );
     }
 
+    const srpmURL = data.url ? (
+        <strong>
+            <a href={data.url}>Downloadable SRPM</a>
+        </strong>
+    ) : (
+        <a></a>
```
Comment (TomasTomecek) [inline frontend/components/results_srpm.js:68]: I'm wondering if we could make the else branch better, maybe "SRPM not available yet"?

Comment (mfocko) [reply frontend/components/results_srpm.js]: I'm actually thinking about moving it up (above the logs)

**Comment by mfocko** [comment]
Looks something like this at the latest commit:
image

**Review by TomasTomecek — APPROVED** [review]
a neat improvement!

**Pull request merged**